### PR TITLE
Apply VERF patch

### LINKED ISSUE #12: Unpublished Branches show "Restricted Access" in "Blog Posts Listing" Block
Observed behavior:

- place the "Blog posts listing" Paragraph
- unpublish a Branch node
- view "Restricted Access" in Location dropdown 
Blog___YMCA_of_Bucks_County

Expected behavior:

- unpublished Branches/Camps should not show up at all in the filter

This is a known issue with VERF, we just need to apply https://www.drupal.org/project/verf/issues/3100599#comment-14219091

### PR DESCRIPTION
**Pull request by froboy**
resolves #12

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] see #12 

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -235,6 +235,9 @@
             },
             "drupal/easy_breadcrumb": {
                 "Fix PHP 8.1 deprecations": "https://git.drupalcode.org/project/easy_breadcrumb/-/merge_requests/42.patch"
+            },
+            "drupal/verf": {
+                "https://dgo.re/3100599 Unpublished content should show nothing in options instead of showing Restricted Access": "https://www.drupal.org/files/issues/2021-09-15/verf-add-access-control-3100599-16.patch"
```
Comment (podarok) [inline composer.json:240]: https://www.drupal.org/project/verf/releases/2.0.0

**Review by podarok — CHANGES_REQUESTED** [review]
Let's use version 2

File: composer.json [hunk 2]
```diff
@@ -238,6 +238,9 @@
             },
             "drupal/video_embed_field": {
                 "Issue #3215817: Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.": "https://git.drupalcode.org/project/video_embed_field/-/merge_requests/3.diff"
+            },
+            "drupal/easy_breadcrumb": {
```
Comment (froboy) [inline composer.json:242]: Not sure why this is showing as added, but i think it should be there as per https://github.com/ymcatwincities/openy/commit/44a6074b392c06fe88093bb0f1dab5537c787648

**Comment by podarok** [comment]
All good 
image
Thank you @froboy

**Pull request merged**